src/main/fc/fc_msp.c

```diff
@@ -822,6 +823,20 @@ static bool mspFcProcessOutCommand(uint8_t cmdMSP, sbuf_t *dst, mspPostProcessFn
         }
         break;
 
+#ifdef VTX_SMARTAUDIO
```

Comment:
This needs to be consolidated with RTC6705 VTX, rather than done on a build flag.